### PR TITLE
completed my home page

### PR DESCRIPTION
**Pull request by GarvG**
Completed my Home page for the CL PLease review My PR

### CODE REVIEW

File: components.css [hunk 1]
```diff
@@ -66,7 +66,7 @@ ul li:hover a{
     text-align: center;
     font-weight: 700;
     background-color: #FFDEFA;
-    height: 60px;
+    height: 4.4rem;
```
Comment (Shubhamkr2610) [inline components.css:69]: sugesstion: use color with variable name

File: components.css [hunk 2]
```diff
@@ -66,7 +66,7 @@ ul li:hover a{
...
     padding-top: 10px;
     font-size: 1.5rem;  
     padding-left: 250px;
```
Comment (Shubhamkr2610) [inline components.css]: suggestion: use rem instead of px for more than 20px

**Review by Shubhamkr2610 — APPROVED** [review]
made the changes that I have suggested . rest good to merge

**Review by humzahasan — APPROVED** [review]
As @Shubhamkr2610 mentioned you can work on the same points, in addition to this.

UI Suggestion: the font wieight of the text on your homepage can be increased abit. Would look better and more readable.

**Pull request merged**